tools/ecfr-parser/main.go

```diff
@@ -270,6 +272,25 @@ func handlePart(thread int, ctx context.Context, date time.Time, reg *eregs.Part
 		return err
 	}
 
+	log.Debug("[worker ", thread, "] Extracting supplemental content structure ", reg.Name, " version ", reg.Date)
+	supplementalPart, err := ecfr.ExtractStructure(*reg.Structure)
+	if err != nil {
+		return err
+	}
+
+	log.Debug("[worker ", thread, "] Posting supplemental content structure ", reg.Name, " version ", reg.Date, " to eRegs")
+	suppResp, err := eregs.PostSupplementalPart(ctx, supplementalPart)
+	if err != nil {
+		if suppResp != nil {
+			defer suppResp.Body.Close()
+			response, e := io.ReadAll(suppResp.Body)
+			if e != nil {
+				log.Println("[ERROR], e")
+			}
+			return fmt.Errorf("%s | %s", err.Error(), string(response))
+		}
+		return err
+	}
```

Comment:
Same as above, consider replacing with just:
```
if err := eregs.PostSupplementalPart(ctx, supplementalPart); err != nil {
   return err
}
```
avoids the potential double-call of `resp.Body.Close()`